docs/platforms/javascript/common/user-feedback/index.mdx

```diff
@@ -28,16 +28,22 @@ The embeddable JavaScript widget allows users to submit feedback from anywhere i
 
 To set up the integration, add the following to your Sentry initialization. There are many options you can pass to the integration constructor. See the [configuration documentation](/platforms/javascript/user-feedback/configuration/) for more details.
 
-### Session Replay
-
-The User Feedback Widget integrates easily with <PlatformLink to="/session-replay/">Session Replay</PlatformLink>. First, make sure that the Session Replay integration is configured correctly and that `replaysOnErrorSampleRate` is greater than 0. When this is done, the Replay SDK will buffer up to 30 seconds worth of events until the user *opens* the User Feedback Widget. If the user submits feedback, you'll be able to view the Feedback (including the replay), inside of Sentry's web application.
-
 <SignInNote />
 
 <PlatformContent includePath="user-feedback/setup" />
 
 By default, this will insert the widget into the bottom right corner of your website. You're free to customize nearly every aspect of the widget, including replacing it completely with your own UI.
 
+On SDK version 8.0.0 and above, users can send screenshots with their feedback. You can configure this using the `showScreenshot` option, by default it is set to `true`. Screenshots aren't supported on mobile devices, so the screenshot button will be hidden automatically in this case.
```

Comment:
can we mention here (and on the config page above too) the min. version for on-prem users to have that enables screenshot support?

Reply:
We've got:
- SaaS -> Software as a service, sentry.io where people signup and login on their own. There's a single database will all the orgs and users and so on saved.
- ST -> Single Tenant, a separate deployment of sentry that we (the company) manage. We run the servers, but the servers only store information for one company.
- On-Prem -> Someone downloads sentry from github and runs it themselves. This'll also (almost always) just have 1 org saved into the database.

S4S is an example of ST where we can deploy the sentry codebase and not affect sentry.io. ST and On-Prem have very similar configs, while the SaaS codebase is special because it has the extra billing layer added on top (all the features are the same everywhere though)

Reply:
Updated! Feel free to make a suggestion if the wording is iffy